.github/workflows/run-orch-smoketest.yml

```
@@ -30,6 +31,46 @@ jobs:
                                   -s secret/kdux/scp/staging/scp_service_account.json \
                                   -e test -v $DOCKER_IMAGE_TAG -n single-cell-portal-test \
                                   -c bin/run_orch_smoke_test.sh
+      - name: Notify Slack failure
+        id: notify-slack-on-fail
+        uses: slackapi/slack-github-action@v1.24.0
+        if: failure()
+        with:
+          channel-id: "C05JHJ0UA1M" #scp-alerts
+          payload: |
+            {
+              "blocks": [
+                { 
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "@channel :x: Orchestration smoke test: ${{ job.status }}\nhttps://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

If we go with [failure-only alerts](https://github.com/broadinstitute/single_cell_portal_core/pull/1857/files#r1298605498):

```suggestion
                    "text": ":x: Orchestration smoke test: ${{ job.status }}\nhttps://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```